app/src/common/components/Button.tsx

```diff
@@ -32,7 +32,7 @@ export const EditedButton = withStyles((theme) => ({
     },
   },
   disabled: {
-    border: "none",
+    border: "1px solid transparent",
```

Comment:
When I added a border to the buttons a long time ago, it was added in outline, instead of inline. So when you go from disabled to enabled, the button changes its size. And when you put a button without a border I added, it's not the same size.
I added to this change a change in the size of the buttons (min height to small, large, and medium), and I only applied the border to the contained AND disabled elements, as the others have no borders added when they are able.